thumbor/config.py

```diff
@@ -267,6 +267,7 @@
         'thumbor.filters.no_upscale',
         'thumbor.filters.saturation',
         'thumbor.filters.max_age',
+        'thumbor.filters.curve'
```

Comment:
I think there should be a trailing comma here so future changes won't cause a diff on this line.